code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -385,7 +385,7 @@
 	for(var/i in 1 to 6)
 		new /obj/item/ammo_box/magazine/m12g(src)
 	new /obj/item/ammo_box/magazine/m12g/buckshot(src)
-	new /obj/item/ammo_box/magazine/m12g/stun(src)
+	new /obj/item/ammo_box/magazine/m12g/slug(src)

```

Comment:
@MMMiracles wanted taser rounds to be the default ammo, so I had to make slugs the child magazine type so that it can still be loaded. I swapped all the /stun to m12g, and all the m12g to /slug